CMakeLists.txt

```diff
@@ -15,6 +15,14 @@ set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+if(APPLE AND EXISTS /usr/local/opt/qt@5.5)
+    # This project requires qt v5.5.
+    # On Mac, Homebrew installs this version in /usr/local/opt/qt@5.5
+    # without this it looks in the usual /usr/local prefix.
+    # this must occur before the find_package call below
+    list(APPEND CMAKE_PREFIX_PATH "/usr/local/opt/qt@5.5")
```

Comment:
I would prefer adding in the building instructions: "use -DCMAKE_PREFIX_PATH=/usr/local/opt/qt@5.5" rather than hard-coding paths in cmakelists. If Qt is installed any other way, that will break.

Travis uses "CMAKE_PREFIX_PATH=/usr/local/opt/qt" without the version number.

Reply:
So, I'm not familiar with cmake, I don't use it on the reg. Would you be fine if this path didn't include the version number + the instructions indicated that it can be overridden?

Reply:
I was suggesting to not change CMakeLists.txt and write something in the building instructions like:
```
brew install qt@5.5
export CMAKE_PREFIX_PATH=/usr/local/opt/qt@5.5
...
```

> Would you be fine if this path didn't include the version number?

It would be fine if it was a standard location. If you install qt@5.x (specifying a version number), will /usr/local/opt/qt work? If it is too hard to predict where qt-sdk is installed. I think it would be better for the user to set it explicitly, this way it is easier for the user to adapt the value if cmake fails to find Qt.